packages/orbit-components/src/Popover/Popover.stories.tsx

```diff
@@ -181,7 +177,7 @@ export const OpenedByProp: Story = {
 export const Overlapped: Story = {
```

Comment:
Please, take a look at this role issue in SB. It's in every story where `controls` are not disabled.

<img width="489" alt="image" src="https://github.com/user-attachments/assets/78895ef8-1df7-423a-a664-5c46473bedd2" />

Reply:
Because at first I had `role` as a string. I will change